examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -63,10 +63,35 @@ parameters:
 
 * `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. See vSphere docs for details. Default: `"thin"`.
 
+#### Portworx Volume
+
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: portworx-io-priority-high
+provisioner: kubernetes.io/portworx-volume
+parameters:
+  repl: "1"
+  snap_interval:   "70"
+  io_priority:  "high"
+
+```
+
+*  `fs`: filesystem to be laid out: none|xfs|ext4 (default: `ext4`)
+*  `block_size`: block size in Kbytes (default: `32`)
+*  `repl`: replication factor [1..3] (default: `1`)
+*  `io_priority`: IO Priority: [high|medium|low] (default: `low`)
+*  `snap_interval`: snapshot interval in minutes, 0 disables snaps (default: `0`)
+*  `aggregation_level`: specifies the number of parts the volume can be aggregated from (default: `0`)
```

Comment:
Specifies the number of replication sets the volume can be aggregated from (default: `1`)